### PR TITLE
Adding netgen package.

### PR DESCRIPTION
**Pull request by mithro**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +25,12 @@ jobs:
     os: linux
     dist: xenial
     env:
-    - PACKAGE=misc/magic
+    - PACKAGE=misc/netgen
```
Comment (PiotrZierhoffer) [inline .travis.yml:28]: Is this a leftover?

File: misc/netgen/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - tcsh
+    - cairo
+    - m4
+    - tk
+    - ncurses
+    - python 3.7
```
Comment (PiotrZierhoffer) [inline misc/netgen/meta.yaml:33]: As reported by GLatosinski regarding the magic package, this should probably be `python>=3.7`. Skywater PDK relies on Python 3.8.

File: misc/netgen/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - readline
+    - fontconfig
+    - xorg-libx11
+  run:
+    - python 3.7
```
Comment (PiotrZierhoffer) [inline misc/netgen/meta.yaml:38]: same here, I think?

**Review by PiotrZierhoffer — CHANGES_REQUESTED** [review]

**Comment by PiotrZierhoffer** [comment]
@mithro we can implement my remarks, but would like your comment on them

File: misc/netgen/meta.yaml (hunk 2 above)
Comment (mithro) [reply misc/netgen/meta.yaml]: Fixed.

File: misc/netgen/meta.yaml (hunk 3 above)
Comment (mithro) [reply misc/netgen/meta.yaml]: Fixed.

File: .travis.yml (hunk 1 above)
Comment (mithro) [reply .travis.yml]: Fixed.

**Comment by mithro** [comment]
All you changes should be fixed now.

**Comment by ajelinski** [comment]
I messed up and instead of pushing rebased branch to `mithro:master` I pushed current `master` there.

Since this PR has been closed automatically and I can no longer push to `mithro:master` I've pushed the rebased branch as `litex-hub:mithro-master` and created a new PR: https://github.com/litex-hub/litex-conda-eda/pull/63

**Pull request closed without merging**